frontend/rust-lib/flowy-folder2/src/entities/view.rs

```diff
@@ -87,6 +91,7 @@ pub fn view_pb_with_child_views(view: Arc<View>, child_views: Vec<Arc<View>>) ->
     layout: view.layout.clone().into(),
     icon_url: view.icon_url.clone(),
     cover_url: view.cover_url.clone(),
+    is_favorite: false,
```

Comment:
```suggestion
    is_favorite: view.is_favorite,
```

Reply:
This helped me figure out a major bug, thanks!